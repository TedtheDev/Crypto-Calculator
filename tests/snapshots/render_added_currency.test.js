import React from 'react';
import AddedCurrency from '../../src/components/added_currency';
import renderer from 'react-test-renderer'

test('<AddedCurrency /> renders correctly', () => {
    const component = renderer.create(
        <AddedCurrency 
            coin={{id: 'btc', name: 'Bitcoin', price_usd: 100}}
            amountOwned={5}
            onChangeAmountOwned={() => {}}
            removeCurrency={() => {}}
        />
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot;
})