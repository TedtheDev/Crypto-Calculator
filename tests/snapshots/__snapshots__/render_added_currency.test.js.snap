// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AddedCurrency /> renders correctly 1`] = `
Array [
  <div>
    Bitcoin
  </div>,
  <div
    className="ui input"
  >
    <input
      disabled={undefined}
      onChange={[Function]}
      placeholder="Enter Amount Owned"
      tabIndex={undefined}
      type="text"
      value="5"
    />
  </div>,
  <div>
    100
  </div>,
  <div>
    =
  </div>,
  <div>
    <div>
      500
    </div>
    <button
      onClick={[Function]}
    >
      Remove
    </button>
  </div>,
]
`;
